polar_route/vessel_performance/vessels/abstract_plane.py

```diff
@@ -0,0 +1,138 @@
+from polar_route.vessel_performance.abstract_vessel import AbstractVessel
+from meshiphi.mesh_generation.aggregated_cellbox import AggregatedCellBox
+from abc import abstractmethod
+import logging
+
+
+class AbstractPlane(AbstractVessel):
+    """
+        Abstract class to model the performance of an underwater glider
+    """
+    def __init__(self, params):
+        """
+            Args:
+                params (dict): vessel parameters from the vessel config file
+        """
+        self.vessel_params = params
+        logging.info(f"Initialising a vessel object of type: {self.vessel_params['vessel_type']}")
+        self.max_speed      = self.vessel_params['max_speed']
+        self.speed_unit     = self.vessel_params['unit']
+        self.max_elevation  = -1 * self.vessel_params['min_depth']
```

Comment:
Remove this factor of -1 for planes/aerial drones and just specify max elevation in the config